README.md

```diff
@@ -1,14 +1,14 @@
 # Universal Android Debloater GUI
-**DISCLAIMER**: Use at your own risk. I am not responsible for anything that could happen to your phone.
+**DISCLAIMER**: Use at your own risk. I am not responsible for anything that could happen to your device.
 
 <img src="/resources/screenshots/v0.5.0.png" width="850" alt="uad_screenshot">
 
 **This software is still in an early stage of development. Check out the issues, and feel free to contribute!**
 
 ## Summary
-This is a complete rewrite in Rust of the [UAD project](https://gitlab.com/W1nst0n/universal-android-debloater), which aims to improve privacy and battery performance by removing unnecessary and obscure system apps. This can also contribute to improve security by reducing [the attack surface](https://en.wikipedia.org/wiki/Attack_surface).
+This is a complete rewrite in Rust of the [UAD project](https://gitlab.com/W1nst0n/universal-android-debloater), which aims to improve privacy and system performance (energy, speed, & memory) by removing unnecessary and obscure system apps. This can also contribute to improve security by reducing [the attack surface](https://en.wikipedia.org/wiki/Attack_surface).
```

Comment:
```suggestion
This is a complete rewrite in Rust of the [UAD project](https://gitlab.com/W1nst0n/universal-android-debloater), which aims to improve privacy and system performance (battery, speed, & memory) by removing unnecessary and obscure system apps. This can also contribute to improve security by reducing [the attack surface](https://en.wikipedia.org/wiki/Attack_surface).
```

Reply:
"energy" already includes "battery" and "external power". Some people also care about general energy savings, not just battery

Reply:
> "energy" already includes "battery" and "external power". Some people also care about general energy savings, not just battery

I don't really agree. "Energy" is super generic and what does "external power" mean in this context and how is it relevant? The only thing that is relevant is battery consumption. Most people wouldn't know what you mean with "energy", as it's too generic. Using "battery" makes a lot more sense to most people, I think.

Reply:
> "Energy" is super generic

That was my intention, to make it generic, so that it matches any kind of device.

> and what does "external power" mean in this context and how is it relevant?

USB, AC, DC, Wireless with base, Truly Wireless (EM waves), etc. (any other past, present, or future, way of transferring any kind of energy to a device). Nowadays, there are many alternatives for power transfer (perhaps we should use "power" rather than "energy"?)

> What kind of energy savings? The only thing that is relevant is battery consumption.

In most cases, yes. But there's also an economic aspect. Some people may be too poor to pay for energy (usually electricity), so they want to save as much money as possible. 

> Most people wouldn't know what you mean with "energy", as it's too generic.

I think I know what you mean 🤔. Since "energy" can be anything, but "battery" usually means "electrochemical energy storage", "battery" is more specific for mobile devices. But "battery" has the same problem, since there are kinetic, chemical, thermal, radioactive, hydraulic, magnetic, photonic, etc... batteries.

However, I agree that (informally) "battery" means "electric storage", so it makes sense to use that word

Reply:
> That was my intention, to make it generic, so that it matches any kind of device.

You can phrase it in a different way, so that it covers all devices, but is still somewhat relevant instead of using 'energy'.

> USB, AC, DC, Wireless with base, Truly Wireless (EM waves), etc. (any other past, present, or future, way of transferring any kind of energy to a device). Nowadays, there are many alternatives for power transfer (perhaps we should use "power" rather than "energy"?)

Lol, this is really far-fetched. If you're debloating your phone, how is it going to be relevant to USB, AC, DC and other radio frequencies such as Wi-Fi and 4G? It makes no sense to use 'energy' here. People will get confused, so instead 'battery life' should be used. Everyone is familiar with that term and knows what it means. 'Energy' is just way too generic and doesn't say anything about what you mean, unless you explain its context.

> In most cases, yes. But there's also an economic aspect. Some people may be too poor to pay for energy (usually electricity), so they want to save as much money as possible.

What? This is super far-fetched, too. As if people are going to use a computer (which uses a lot of power in comparison to a phone) to use UAD, which in turn saves marginal what? Charging your battery a little less frequent? People who are too poor to pay for electricity aren't the people who use a PC (that in comparison drains *much* more) to debloat their phone, lol.

> I think I know what you mean 🤔. Since "energy" can be anything, but "battery" usually means "electrochemical energy storage", "battery" is more specific for mobile devices. But "battery" has the same problem, since there are kinetic, chemical, thermal, radioactive, hydraulic, magnetic, photonic, etc... batteries.

> However, I agree that (informally) "battery" means "electric storage", so it makes sense to use that word

You're thinking way too difficult I think. I think instead using 'battery life' is better, because everyone knows what it means without having to explain context (see an earlier reply in this comment).

Reply:
> You can phrase it in a different way

IDK how 😅

> If you're debloating your phone, how is it going to be relevant to USB, AC, DC and other radio frequencies such as Wi-Fi and 4G? It makes no sense to use 'energy' here.

Good point. But (for the record), I didn't mean that "debloating magically reduces wave interference". EM charging rarely uses the same frequency bands as Bluetooth, WiFi, or cellular data.
 
> What? This is super far-fetched, too. As if people are going to use a computer (which uses a lot of power in comparison to a phone) to use UAD, which in turn saves marginal what? Charging your battery a little less frequent? People who are too poor to pay for electricity aren't the people who use a PC (that in comparison drains _much_ more) to debloat their phone, lol.

I feel roasted, LMAO. But it's fine. You're right.

> You're thinking way too difficult I think.

Yeah, I tend to think too much sometimes 😅

> I think instead using 'battery life' is better, because everyone knows what it means without having to explain context

I'm still not sure (but you're on the right track! I also want to make the `README` easier to understand). @0x192 , what do you think? 👀

Reply:
Sorry if I came off harsh, I didn't mean to.

> IDK how 😅

I think using 'battery life' is good enough to be honest.

> Good point. But (for the record), I didn't mean that "debloating magically reduces wave interference". EM charging rarely uses the same frequency bands as Bluetooth, WiFi, or cellular data.

Yeah I know, but I don't see how it is related to battery drain, and if it is, it'd be marginal.

Reply:
> Sorry if I came off harsh, I didn't mean to.

Don't worry :) I'm okay

> I think using 'battery life' is good enough to be honest.

That could be confused with "battery lifespan" and "battery health". Although it improves both, so there's not much of a issue

Reply:
> That could be confused with "battery lifespan" and "battery health". Although it improves both, so there's not much of an issue

Yeah, but in this context most people also know what it means, and as you said: it improves both. Could you accept my suggestion so that it gets committed?

Reply:
I think it's fine. The original text said "battery". We'll stick with the old one, unless @0x192 says otherwise